server/scsynth/SC_PortAudio.cpp

```diff
@@ -357,14 +361,23 @@ bool SC_PortAudioDriver::DriverSetup(int* outNumSamples, double* outSampleRate)
             mOutputChannelCount = 0;
         }
 
+#ifdef __APPLE__
+        // use PortAudio setting tuned for pro audio apps
+        PaMacCoreStreamInfo macInfo;
+        PaMacCore_SetupStreamInfo(&macInfo, paMacCorePro);
```

Comment:
from http://portaudio.com/docs/v19-doxydocs/pa__mac__core_8h.html -- `paMacCorePro == paMacCoreChangeDeviceParameters == 0x1`, and the documentation for `paMacCoreChangeDeviceParameters` says

> Allows PortAudio to change things like the device's frame size, which allows for much lower latency, but might disrupt the device if other programs are using it, even when you are just Querying the device. 

is that really what we want?

Reply:
That's what coreaudio backend is doing, AFAIU.